### DISCUSSION TITLE
How to use js pure in the front end

### DISCUSSION THREAD

**Discussion by faisal-almalki** [strapi/strapi#7565] [Legacy Topics]
Hello I'm trying to use strapi with vanilla js for the front end but i don't know how to do it, I see some tutorials and they use vue or react and I want see if there is why to use vanilla js

**Comment by derrickmehaffy** [comment]
I'm not sure I understand the question, what are you referring to by `frontend` in this case?

**Reply by faisal-almalki** [reply]
I see some courses about how to use strapi and they always use react or vue for the front-end but I just want to use vanilla js is, I know this possible but I don't know-how.

like in this course https://blog.logrocket.com/how-to-build-an-ecommerce-site-with-strapi-vue-js-and-flutterwave/ he uses vue I want to use vanilla js

**Reply by derrickmehaffy** [reply]
In this case it might be better to request a tutorial on https://github.com/strapi/community-content
